src/js/firstparties/twitter.js

```diff
@@ -0,0 +1,62 @@
+let query_param = 'data-expanded-url';
+let found_tcos_query = "";
+let fixes = {};
+let tcos_with_target_query = "a[" + query_param + "][href^='https://t.co/'], a[" + query_param + "][href^='http://t.co/']";
+
+function maybeAddNoreferrer(element) {
+  let rel = element.rel ? element.rel : "";
+  if (!rel.includes("noreferrer")) {rel += " noreferrer";}
+  element.rel = rel;
+}
+
+function unwrapTco(tco, target) {
+  if (!target) {
+    return;
+  }
+  tco.href = target;
+  tco.addEventListener("click", function (e) {
+    e.stopPropagation();
+  });
+  maybeAddNoreferrer(tco);
+}
+
+function checkLink(linkElelement) {
+  let attr = linkElelement.getAttribute(query_param);
+  if (attr && (attr.startsWith("https://") || attr.startsWith("http://"))) {
+    let toBeReplaced = linkElelement.href;
+    fixes[toBeReplaced] = attr;
+    if (found_tcos_query) {
+      found_tcos_query += ", ";
+    }
+    unwrapTco(linkElelement, attr);
+    found_tcos_query += "a[href='" + toBeReplaced + "']";
+  }
+}
+
+function unwrapTwitterURLs() {
+  function removeInDoc(doc) {
+    let yes_target = doc.querySelectorAll(tcos_with_target_query);
+    for (let i = 0; i < yes_target.length; i++) {
+      checkLink(yes_target[i]);
+    }
+    if (found_tcos_query) {
+      let no_target = doc.querySelectorAll(found_tcos_query);
+      for (let i = 0; i < no_target.length; i++) {
+        unwrapTco(no_target[i], fixes[no_target[i].href]);
+      }
+    }
+  }
+
+  removeInDoc(document);
+  let iframes = document.getElementsByTagName('iframe');
```

Comment:
Because then those content scripts won't share the global `found_tcos_query` and `fixes` variables that they need to replace links which were found in other iframes.